### PR TITLE
Add address custom field

### PR DESCRIPTION
**Pull request by lxkuz**
Добавление кастомного поля с типом адрес. Первое поле адреса становится стандартным адресом заявки

### CODE REVIEW

File: app/helpers/admin/help_request_kind_helper.rb [hunk 1]
```diff
@@ -6,7 +6,7 @@ def options_for_datatype
         ['Галочка (Да/Нет)', :checkbox],
         ['Текстовое поле с переносами строки и возможностью форматирования', :textarea],
         ['Телефон', :phone],
-        ['Дата и время', :date]
```
Comment (lxkuz) [inline app/helpers/admin/help_request_kind_helper.rb:9]: get it back please

File: app/views/admin/help_requests/_custom_values.html.erb [hunk 2]
```diff
@@ -5,6 +5,7 @@
     <div id="js-custom-fields-container" data-values='<%= @custom_values.to_json %>'>
       <custom-fields-panel 
         :request-path="'<%= custom_fields_admin_help_request_path(f.object.try(:id) || 0) %>'"
+        empty-point-error-message="<%= I18n.t("help_request.errors.empty_point") %>"
```
Comment (lxkuz) [inline app/views/admin/help_requests/_custom_values.html.erb:8]: hmm I guess we could follow [this soultion](https://www.stefanwienert.de/blog/2019/11/02/integrating-javascript-vue-i18n-into-rails-missing-auto-translate-pipeline/) to pass i18n into JS context... but not required

**Review by lxkuz — COMMENTED** [review]
Good job! Apart from the comments, I would ask you to squash my `tmp`-like prev commits, please. Thanks, @StenvL !

**Comment by lxkuz** [comment]
@StenvL  one more thing! to complete this task we also need to render address custom field in API:
1) as a string for API v1/v2
2) as json fro API v3
We need to discuss it

**Review by memumen — APPROVED** [review]

**Pull request merged**